client/src/app/components/pai/pai.component.ts

```diff
@@ -84,7 +84,8 @@ export class PaiComponent implements OnInit, OnDestroy {
 
   loadData(root) {
     const partition = d3partition();
-    const color = scale.scaleOrdinal(scale.schemeCategory20);
+    // TODO Change the schemeCategory20 to the new
```

Comment:
@DavidBronfen We talked about this.
Let's not forget to do this before we merge.